engine/src/juliabox/db/instance.py

```diff
@@ -99,7 +99,7 @@ def get_stale_instances(cluster):
         valid_time = nowsecs - JBoxInstanceProps.SESS_UPDATE_INTERVAL
         stale = []
         for record in JBoxInstanceProps.scan(instance_id__beginswith=cluster, publish_time__lt=valid_time):
-            stale.append(record.get('instance_id'))
+            stale.append(record.get('instance_id').split('.')[1])  # split cluster name from instance name
```

Comment:
Also, to help in migration, when there will be existing instance entries without the qualifier:
````
if record has '.':
    split and take trailing part
else:
    take full record
````